src/common/updates.cpp

```diff
@@ -45,10 +45,7 @@ namespace tools
 
     // All four MoneroPulse domains have DNSSEC on and valid
     static const std::vector<std::string> dns_urls = {
-        "updates.moneropulse.org",
-        "updates.moneropulse.net",
-        "updates.moneropulse.co",
-        "updates.moneropulse.se"
+        "chain.updates.lt.hn"
```

Comment:
@bodane can you sort this one please?

Reply:
Just a dns record I assume?

Reply:
yes, on `rick:lt.hn` if you map it to `download.lt.hn` it will be served from a Global CDN

here is some @letheanVPN/documentation information. 

URL: `download.lt.hn`

Storage locations: ` New York, Los Angeles, Singapore, Sydney`

GFX: <img width="654" alt="image" src="https://user-images.githubusercontent.com/631881/146947252-a90172b0-6bac-4a78-ac69-e642683e45be.png">

POP Count: 74+
Adverage Global Latency: < 29ms
GFX: 
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/631881/146948266-5dc810d5-d7f6-44e4-adb4-04367893f6f6.png">